libraries/aws_iam_root_user.rb

```diff
@@ -0,0 +1,28 @@
+# author: Miles Tjandrawidjaja
+class AwsIamRootUser < Inspec.resource(1)
+  name 'aws_iam_root_user'
+  desc 'Verifies settings for AWS root account'
+  example "
+    describe aws_iam_root_user do
+      its('access_key_count') { should eq 0 }
+    end
+  "
+
+  def initialize(conn = AWSConnection.new)
+    @client = conn.iam_client
+  end
+
+  def access_key_count
+    summary_account['AccountAccessKeysPresent']
+  end
+
+  def to_s
```

Comment:
Yeah I'm happy to change it. Usually where there is a lack of guidelines I like having something that looks not quite right, so it prompts consistency. What is your suggestion, I am thrown off by the "Account ID" comment.

1. AWS Root-User
2. Root-User <AccountID>